### ISSUE TITLE
Questions about workflow and evaluation when setting up validation operations

### ISSUE THREAD

**Issue by yzfzzz** [open-mmlab/mmdetection#9351]
Hello, I would like to ask：

When I want to reason in a test set, I just need to set `evaluation`.

```python
evaluation = dict(interval=1, metric='bbox', classwise=True)
```

When I just want to train and validate at the same time, I just need to set `workflow` without considering `evaluation`: ``` python

```python
workflow = [('train', 1),('val',1)]
```

I don't know if I'm thinking this way correctly, I hope I can get your answer, if I want to validate, how should I modify the config file? (I have the above question because the format of my training validation set, and test set are different)
